src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -1028,6 +1029,15 @@ export const GlobalProvider = memo(
                 });
                 outputDataActions.delete(id);
                 addInputDataChanges();
+                const nodeSchemaId = getNode(id)?.data.schemaId;
+                if (nodeSchemaId && isStartingNode(schemata.get(nodeSchemaId))) {
+                    backend
+                        .clearNodeCacheIndividual(id)
+                        .then(() => {})
```

Comment:
This doesn't do anything and can be removed.

```suggestion
```

Reply:
That's there (as well as in other places) because eslint yells at you if you dont await a promise unless you have a .then and a .catch.

Well, now that I think about it maybe it only needs the catch and I just misread the error.... Will try

Reply:
It only yells at you when you don't handle the error a promise could "throw". So `.then(fn)` doesn't fix the lint, but `.then(fn, fn)` and `.catch(fn)` will (`.catch(fn)` is the same as `.then(x => x, fn)` btw).

Reply:
yeah i definitely misread the error. it must end in a call to .then OR .catch, not both